docs/vsphere-prerequisites.md

```diff
@@ -17,13 +17,16 @@ The follwing section describes the setup of the internal API server endpoint wit
 ## vSphere environment changes
 
 WMCO downloads the ignition files from the internal API server endpoint. To enable this, add a new DNS entry for
-```html
+```
 api-int.<cluster-name>.domain.com
 ```
+
 which points to the external API server URL
-```html
+
+```
 api.<cluster-name>.domain.com
 ```
+
```

Comment:
this commit has some changes unrelated to the commit subject, can that be fixed?

Reply:
Good point, will adjust. Thanks @mansikulkarni96